paper/paper.Rmd

```diff
@@ -42,23 +49,113 @@ link-citations: yes
+
+Lastly, `{datawizard}` also provides a toolbox to create a detailed profile of data properties.
+
+# Features
+
+## Data wrangling
+
+The raw data is rarely in a state that it can be directly fed into a statistical model. It often needs to be modified in various ways. For example, columns need to be renamed and/or reordered, data scattered across multiple tables needs to be joined, certain parts of the data need to be left out, etc. 
+
+`{datawizard}` provides various functions for cleaning and preparing data (see Table 1).
+
+Function           | Operation                             |
+------------------ | --------------------------------------|
+`data_filter()`    | to select only certain *observations* |
+`data_select()`    | to select only a few *attributes*     |
+`data_extract()`   | to extract a single *attribute*       |
+`data_rename()`    | to rename attributes                  |
+`reshape_longer()` | to convert data from wide to long     |
+`reshape_wider()`  | to convert data from long to wide     |
+`data_join()`      | to join two data frames               |
+    ...            |        ...                            |
+
+Table: The table below lists a few key functions offered by *datawizard* for data wrangling. To see the full list, see the package website: <https://easystats.github.io/datawizard/>
+
+We will look at one example function that converts data in wide format to tidy/long format:
+
+```{r}
+stocks <- data.frame(
+  time = as.Date('2009-01-01') + 0:4,
+  X = rnorm(5, 0, 1),
+  Y = rnorm(5, 0, 2)
+)
+
+stocks
+
+data_to_long(
+  stocks,
+  select = -c("time"),
+  colnames_to = "stock",
+  values_to = "price"
+)
+```
+
+## Data transformations
+
+Even after getting the raw data in the needed format, we may further need to transform certain variables further to meet requirements imposed by the statistical model.
+
+`{datawizard}` provides a rich collection of such functions for transforming variables (see Table 2).
+
+Function           | Operation                                     |
+------------------ | ----------------------------------------------|
+`standardize()`    | to center and scale data                      |
+`normalize()`      | to scale variables to 0-1 range               |
+`adjust()`         | to adjust data for effect of other variables  |
+`data_shift()`     | to shift numeric value range                  |
+`ranktransform()`  | to convert numeric values to integer ranks    |
+    ...            |        ...                                    |
+
+Table: The table below lists a few key functions offered by *datawizard* for data transformations. To see the full list, see the package website: <https://easystats.github.io/datawizard/>
```

Comment:
I am deliberately not being comprehensive because we might add more functions in the future and `...` cover all of those.